eccehomo/test_settings.py

```diff
@@ -0,0 +1,4 @@
+import os
+
+TEST_MEDIA_ROOT = os.path.join(os.getcwd(), 'test_media')
+filename = os.path.join(TEST_MEDIA_ROOT, 'test_image.jpg')
```

Comment:
You can avoid having this file using:
- A environment variable override on the `run_tests.py` file 
- Not having this `filename` variable here, setup on the `setUpClass`/`setUp` of your tests.